includes/class-form-entry.php

```diff
@@ -421,4 +421,10 @@ public function get_payment_data() {
 
         return $wpdb->get_row( "SELECT * FROM {$wpdb->prefix}weforms_payments WHERE entry_id = {$this->id} " );
     }
+
+    public static function get_form_id( $entry_id ) {

```

Comment:
MIssing function docblock. When you do add the docblock, be sure to add the @global tag. 
see https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#1-functions-class-methods